hack/e2e-internal/e2e-up.sh

```diff
@@ -31,5 +31,14 @@ source "${KUBE_VERSION_ROOT}/cluster/${KUBERNETES_PROVIDER}/util.sh"
 
 prepare-e2e
 
-"${KUBE_VERSION_ROOT}/cluster/kube-up.sh"
+if [[ ${MULTIZONE:-} == "true" ]]; then
+    for KUBE_GCE_ZONE in $E2E_ZONES
+    do
+	KUBE_GCE_ZONE=${KUBE_GCE_ZONE} KUBE_USE_EXISTING_MASTER=${KUBE_USE_EXISTING_MASTER} "${KUBE_VERSION_ROOT}/cluster/kube-up.sh"
```

Comment:
I think you either need to set `$KUBE_USE_EXISTING_MASTER="false"` or use a default here, as otherwise the first iteration will fail.